airflow/upgrade/rules/import_changes.py

```diff
@@ -113,7 +113,7 @@ def _check_file(file_path):
         with open(file_path, "r") as file:
             content = file.read()
             for change in ImportChangesRule.ALL_CHANGES:
-                if change.old_class in content:
+                if change.old_path in content:
```

Comment:
While the old version of this was subject to false positives, such as throwing a warning on
`from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator` 
due to the match on `KubernetesPodOperator`

I think that this version will produce false negatives, as we are now searching the file for the full import path (`airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator`)

Which means that something like `from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator` won't be caught as there is no exact match.